src/index.js

```diff
@@ -1,13 +1,53 @@
-export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
+'use strict';
+
+import { createUserNumbers } from './check_user_num.js';
+import { resultMessage } from './count.js';
+
+const app = document.querySelector('#app');
+const result = document.querySelector('#result');
+
+// 1. 컴퓨터 난수 생성
+const computerNumberArray = createComputerNumbers();
+console.log(computerNumberArray);
+
+function baseballGame(e) {
+  e.preventDefault();
+  // 2. 만약 사용자 입력값에 오류가 있다면 경고창을 띄우고, 없다면 배열로 받아오기
+  const userNumberArray = createUserNumbers();
+  // 3. 결과 메시지 출력하기
+  const message = resultMessage(userNumberArray, computerNumberArray);
+  result.textContent = message;
+  // 4. 정답을 맞췄다면, 게임을 다시 시작할건지 물어보는 요소 생성
+  if (message === '🎉정답을 맞추셨습니다!🎉') {
+    askRestart();
+  }
 }
+// 5. 이벤트 등록
+app.addEventListener('submit', baseballGame);
 
-// export default class BaseballGame {
-//   play(computerInputNumbers, userInputNumbers) {
-//     return "결과 값 String";
-//   }
-// }
+// 3자리 난수 생성
+function createComputerNumbers() {
+  let set = new Set();
+  while (set.size < 3) {
+    const num = Math.floor(Math.random() * 10);
+    if (num >= 1) {
+      set.add(String(num));
+    }
+  }
+  const computerNumbersArray = [...set];
+  return computerNumbersArray;
```

Comment:
축약이 가능한 부분에서는 적절히 축약해서 표현해주시면 가독성이 좋아질거 같네요!

```suggestion
  return [...set];
```

Reply:
네 ! ㅎㅎ